### PR TITLE
add types param / support landmark.poi

### PR DESCRIPTION
**Pull request by sbma44**
would love eyes on this -- unsure if this constitutes a breaking change or not.

### CODE REVIEW

**Comment by sbma44** [comment]
@tristen added to defaultProps -- just to document: good to merge?

**Comment by tristen** [comment]
Curious why this PR includes the deletion of `example/bundle.js`?

**Comment by tristen** [comment]
One last item is to add a [`propTypes`](https://github.com/mapbox/react-geocoder/blob/landmarkdown/index.js#L36) definition for it as optional

**Comment by sbma44** [comment]
Roger that -- one more look?

File: example/bundle.js [hunk 1]
```diff
@@ -20235,127 +20320,14 @@ function once (fn) {
...
+},{}],167:[function(require,module,exports){
 'use strict';
 
 var xhr = require('xhr');
 
-function search(endpoint, source, accessToken, proximity, bbox, query, callback) {
+function search(endpoint, source, accessToken, proximity, bbox, types, query, callback) {
   var searchTime = new Date();
-  var uri = endpoint + '/geocoding/v5/' + source + '/' + encodeURIComponent(query) + '.json' + '?access_token=' + accessToken + (proximity ? '&proximity=' + proximity : '') + (bbox ? '&bbox=' + bbox : '');
+  var uri = endpoint + '/geocoding/v5/' + source + '/' + encodeURIComponent(query) + '.json' + '?access_token=' + accessToken + (proximity ? '&proximity=' + proximity : '') + (bbox ? '&bbox=' + bbox : '') + (types ? '&types=' + encodeURIComponent(types) : '');
```
Comment (yulolimum) [inline example/bundle.js]: `encodeURIComponent(types)` - does this need to be encoded? The API requires a comma separated list of keywords (e.g. country,region,locality). If you pass that in as a string into the component, this code would produce: `country%2Cregion%2Clocality`. Not encoding would produce the correct result.

...or am i missing something? @sbma44

Comment (sbma44) [inline example/bundle.js]: `country%2Cregion%2Clocality` is correct -- the comma should be URL encoded (in practice both will work right now).

Comment (yulolimum) [inline example/bundle.js]: It was erring for me but now i can't replicate so 👍  . :)

**Comment by sbma44** [comment]
Good to merge here?

**Comment by tristen** [comment]
@sbma44 LGTM!

**Pull request merged**